Deploy/src/sns.tf

```
@@ -1,5 +1,8 @@
 resource "aws_sns_topic" "alert" {
-  name = "${local.prefix}alert"
+  name = "${local.name-prefix}alert"
+  tags = {
+    notification-target = "alerts-training-${var.env}",

```

Any reason you hard coded this?
```suggestion
    notification-target = "alerts-${var.microservice}-${var.env}",

```